packages/eslint-plugin/src/util/objectIterators.ts

```diff
@@ -8,22 +8,22 @@ function objectForEachKey<T extends Record<string, unknown>>(
   }
 }
 
-function objectMapKey<T extends Record<string, unknown>, TReturn>(
+function objectMapKey<T extends Record<string, unknown>, Return>(
   obj: T,
-  callback: (key: keyof T) => TReturn,
-): TReturn[] {
-  const values: TReturn[] = [];
+  callback: (key: keyof T) => Return,
+): Return[] {
+  const values: Return[] = [];
```

Comment:
@JoshuaKGoldberg I could potentially write a test for this?

Reply:
Also checked to see if `objectMapKey` is used anywhere else and it doesn't seem like it? `yarn test` passed when this function is removed from export.